### PR TITLE
Insourcing `react-toast-notification`

### PR DESCRIPTION
**Pull request by youchann**
ref: https://github.com/voyagegroup/ingred-ui/issues/439
ref: https://github.com/jossmac/react-toast-notifications

> This was a great project to learn from and fulfilled the requirements it set out to. Unfortunately, I can no-longer give this project the time it needs. Consider react-hot-toast as an alternative, or look at the source and make your own 🎉 (there really isn't much to it).

Sooner or later, we have to Insourcing `react-toast-notification`.
So We cloned that library and tweak it for ingred-ui.

### CODE REVIEW

**Comment by youchann** [comment]
### やったこと

- TypeScript化
- `emotion`ではなく`styled-component` でスタイリング

**Comment by youchann** [comment]
@kohashi 
一旦、ソースコードをガッと持ってきましたmm

File: src/lib/react-toast-notification/README.md [hunk 1]
```diff
@@ -0,0 +1,3 @@
+[react-toast-notifications](https://github.com/jossmac/react-toast-notifications) is no-longer maintained.
+
+So We cloned that library and tweak it for ingred-ui.
```
Comment (kohashi) [inline src/lib/react-toast-notification/README.md:3]: 👍 
ライセンス面もよさそう

**Review by kohashi — APPROVED** [review]
移植良さそうです！

**Comment by youchann** [comment]
ローカルで成果物を検証済み

**Pull request merged**